### PR TITLE
[RU]Remanga fix 403 again

### PR DESCRIPTION
**Pull request by e-shl**
Closes https://github.com/tachiyomiorg/tachiyomi-extensions/issues/15868
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: src/ru/remanga/src/eu/kanade/tachiyomi/extension/ru/remanga/Remanga.kt [hunk 1]
```diff
@@ -886,8 +900,7 @@ class Remanga : ConfigurableSource, HttpSource() {
             setDefaultValue(false)
 
             setOnPreferenceChangeListener { _, newValue ->
-                val checkValue = newValue as Boolean
-                preferences.edit().putBoolean(key, checkValue).commit()
+                true
             }
```
Comment (stevenyomi) [inline src/ru/remanga/src/eu/kanade/tachiyomi/extension/ru/remanga/Remanga.kt:904]: Please remove the `{ true }` blocks entirely.

Comment (e-shl) [reply src/ru/remanga/src/eu/kanade/tachiyomi/extension/ru/remanga/Remanga.kt:904]: ?

**Comment by e-shl** [comment]
> [settings hiding under ⚙ in basic tachi](https://github.com/tachiyomiorg/tachiyomi-extensions/pull/15883/commits/f2235a07db678879378226ea68c8f956ccfd2d5b)
for some unknown reason

**Comment by stevenyomi** [comment]
> no one understands what the settings hiding under ⚙ in basic tachi

Because one extension can have multiple sources, and each one have individual settings.

Currently there's another entry point: Source → (some source) → overflow menu (⋮) → Settings.

**Pull request merged**